.circleci/config.yml

```diff
@@ -37,6 +38,19 @@ jobs:
             - ~/.cache/yarn
             - .yarn-cache
             - node_modules
+
+      - restore_cache:
+          keys:
+            - v1-json-{{ .Environment.CIRCLE_SHA1 }}
```

Comment:
This is great for rerunning a build on the same commit which does not happen that often.
We will see how useful the cache is with different builds in different branches.

Reply:
Oops, good catch, should include `CIRCLE_BRANCH`, so there will be more cache hits